mysql-test/main/max_used_connections_time.test

```diff
@@ -0,0 +1,35 @@
+-- source include/not_embedded.inc
+
+--echo #
+--echo # MDEV-30543 New status variable: Max_used_connections_time
+--echo #
+
+FLUSH STATUS;
+--disable_query_log
+--disable_result_log
+connect (con1,localhost,root,,);
+--sleep 1
+connect (con2,localhost,root,,);
+--sleep 1
+let $time_1=`SELECT VARIABLE_VALUE FROM INFORMATION_SCHEMA.SESSION_STATUS WHERE VARIABLE_NAME LIKE 'max_used_connections_time'`;
+--sleep 1
+connection con1;
+disconnect con2;
+--sleep 1
+let $time_2=`SELECT VARIABLE_VALUE FROM INFORMATION_SCHEMA.SESSION_STATUS WHERE VARIABLE_NAME LIKE 'max_used_connections_time'`;
+--echo # This should report 0 as it must have the same value as above
+let $dif_1= `SELECT '$time_1' <> '$time_2'`;
+--echo $dif_1
+--sleep 1
+connect (con3,localhost,root,,);
+--sleep 1
+connect (con4,localhost,root,,);
+--sleep 1
+let $time_3=`SELECT VARIABLE_VALUE FROM INFORMATION_SCHEMA.SESSION_STATUS WHERE VARIABLE_NAME LIKE 'max_used_connections_time'`;
+--echo # This should report 1 as connection number was increased
+let $dif_2= `SELECT '$time_1' <> '$time_3'`;
+--echo $dif_2
+
```

Comment:
Please add `show status` query example.

Reply:
I suggested to @Chaloff that he should change this to `--eval SELECT '$time_3' > '$time_1' as expect_1`, so that the expected value of the comparison is apparent in _both_ the `.test` and the `.result` files.

.test will have:
```
--replace_regex /…/TIMESTAMP/
--eval SELECT '$time_3' > '$time_1' as expect_1
```

.result will have:
```
SELECT 'TIMESTAMP' > 'TIMESTAMP' as expect_1
expect_1
1
```

Reply:
Yes, thanks - will update PR shortly